namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-subject-access-request-dev/resources/irsa.tf

```diff
@@ -0,0 +1,47 @@
+# Add the names of the SQS which the app needs permissions to access.
+# The value of each item should be the namespace where the SQS was created.
+# This information is used to collect the IAM policies which are used by the IRSA module.
+locals {
+  # The names of the queues used and the namespace which created them
+  sqs_queues = {
+    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev"
```

Comment:
This queue name is a bit messy with some capitalised and some not, and both - and _
Is this pattern a convention used elsewhere?

Reply:
The queue name is generated by Audit team, we do not generate the queue and using existing queue